### ISSUE TITLE
[$100] No applicable method for REGION-CONTAINS-POSITION-P with #<DREI:DREI-AREA TEXTUAL-DREI-SYNTAX-VIEW>

### ISSUE THREAD

**Issue by abridgewater** [McCLIM/McCLIM#380]
To reproduce:

1. Start a fresh listener.
2. Issue the Show Used Packages command (either through the command parser or the menu).
3. Hit Enter when prompted for arguments.
4. Behold the NO-APPLICABLE-METHOD.

**Comment by christophejunke** [comment]
I stumbled upon this bug. I have a very shallow understanding of McCLIM, but wouldn't this work?

```
(defmethod region-contains-position-p ((drei drei) x y)
  (region-contains-position-p
   (pane-viewport-region (editor-pane drei)) x y))
```

**Comment by nsrahmad** [comment]
@dkochmanski fixed with 08e3f828a442e7 by @slyrus

**Comment by dkochmanski** [comment]
I'm claiming the issue and re-adding it to the pool.
